src/iOS.Core/Views/FormEntryTableViewCell.cs

```diff
@@ -12,14 +12,15 @@ public class FormEntryTableViewCell : ExtendedUITableViewCell, ISelectable
         public UITextField TextField { get; set; }
         public UITextView TextView { get; set; }
         public UIButton Button { get; set; }
+        public UIButton SecondButton { get; set; }
         public event EventHandler ValueChanged;
 
-
         public FormEntryTableViewCell(
             string labelName = null,
             bool useTextView = false,
             nfloat? height = null,
             bool useButton = false,
+            bool useTwoButtons = false,
```

Comment:
Change this to an enum internal to this class:
```c#
public enum ButtonsConfig : byte
{
     None = 0,
     One = 1,
     Two = 2
}
```
So we avoid the flags, if not it's kind of confusing to have possibilities to pass parameters like `useButton=false` and `useTwoButtons=true` unless you throw an exception if that happens or something like that.
However, I think that the enum way is clearer for usage